### DISCUSSION TITLE
mod_BalloonNotification

### DISCUSSION THREAD

**Discussion by Michael-B-Mueller** [ViperSRT3g/VBA-General#7] [Q&A]
Hi there,

is there any change of reacting to a click on the Notification itself, but not the tray icon.
Currently it's just closing the notification, but I would rather open a third party app.

Thanks
Michi

https://github.com/ViperSRT3g/General-VBA/blob/main/mod_BalloonNotification.bas

**Comment by ViperSRT3g** [comment]
Sorry for missing this! I'd totally forgot about this notification when I first saw it. I can look into adding response options to this, though it may take a while to implement. Currently in a time crunch on .NET things for most of the next month or so.

**Reply by Michael-B-Mueller** [reply]
Oh, thank you for your quick reply.
It would just be nice to have this functionality, if it doesn't work it's not bad. I am already glad to have come across your compact solution for the BalloonTips ;-)

**Reply by ViperSRT3g** [reply]
Haha, no problem. And since you brought up this module, I've already spotted some code improvements I could implement. So at the very least there will be some code improvements on the way once I have the time to start digging into this thing.
